### ISSUE TITLE
What does id in jsx field exactly indicate?

### ISSUE THREAD

**Issue by tippy33** [swsnu/swppfall2021#61]
Hello, I have a question about jsx field id. 
In the instructions on hw3, there are ids for each element in accordance with jsx tag(e.g. jsx tag type: button, id: create-article-button).
Does this id mean the parameter we put when declaring react elements, for example, <button id="back-create-article-button" ... /> ?
Or does it indicate something else?
I was curious because I thought we didn't put id as parameter during our practice sessions. 
Thank you.

**Comment by kdh0102** [comment]
Yes, please implement as you noted in your example, `<button id="back-create-article-button" ... /> `.
